rust/src/config.rs

```diff
@@ -495,4 +498,13 @@ pub const CLIOPTIONS: [CliOption; 16] = [
 		example: "--debug-level 2",
 		kind: OptionType::Number,
 	},
+	CliOption {
+		key: "raw_mode",
+		name: "--raw-mode",
+		shortcut: "-r",
+		fallback_shortcut: "",
+		description: "Use to enable proper newline rendering for 'raw mode' in terminals",
```

Comment:
I'd probably add that we add `\r` to line breaks? What do you think? (changes here should be reflected in the node help as well)

Reply:
Great idea! Will add that